tests/code_size/hello_webgl2_fastcomp_asmjs.json

```diff
@@ -0,0 +1 @@
+{"a.html.gz": 382, "a.mem.gz": 219, "total_gz": 6817, "a.js.gz": 2516, "a.js": 5350, "a.asm.js.gz": 3700, "a.asm.js": 10589, "a.html": 580, "a.mem": 321, "total": 16840}
```

Comment:
can we add newlines here?

Also formatting the json with one line per key would make reading diffs much easier when these get updated.